node/modules/lp2p/host.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/filecoin-project/lotus/node/modules/helpers"
 )
 
+const KadProtocolID = "/fil/kad/1.0.0"
```

Comment:
From [spec](https://filecoin-project.github.io/specs/#systems__filecoin_nodes__network) this should include network name instead of protocol version.